assets/src/blocks/Splittwocolumns/SplittwocolumnsSettings.js

```diff
@@ -1,15 +1,21 @@
+/* eslint-disable jsdoc/require-param-type */
 import { FocalPointPicker, PanelBody, SelectControl } from '@wordpress/components';
 import { InspectorControls } from '@wordpress/block-editor';
 import { ImageOrButton } from '../../components/ImageOrButton/ImageOrButton';
-import { URLInput } from "../../components/URLInput/URLInput";
+import { URLInput } from '../../components/URLInput/URLInput';
 
 const { useSelect } = wp.data;
 const { __ } = wp.i18n;
 
 /**
  * Sidebar settings
+ *
+ * @param  root0
```

Comment:
Here again I think we can remove those `root0` params

Reply:
👍